SoftLayer/CLI/block/subnets/assign.py

```diff
@@ -12,7 +12,13 @@
               help="ID of the subnets to assign; e.g.: --subnet-id 1234")
 @environment.pass_env
 def cli(env, access_id, subnet_id):
-    """Assign block storage subnets to the given host id.
+    """EXAMPLE:
+
+                slcli block subnets-assign 111111 --subnet-id 222222
+                slcli block subnets-assign 111111 --subnet-id 222222 --subnet-id 333333
+                ACCESS_ID is the host_id obtained by: softlayer slcli block access-list <volume_id>
+
+    Assign block storage subnets to the given host id.
```

Comment:
The short description should be first.
EXAMPLE should have 2 colons (`::`), this is a special notation for the documentation ( https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks)
```suggestion
    """Assign block storage subnets to the given host id.
    
    EXAMPLE::

                slcli block subnets-assign 111111 --subnet-id 222222
                slcli block subnets-assign 111111 --subnet-id 222222 --subnet-id 333333
                ACCESS_ID is the host_id obtained by: softlayer slcli block access-list <volume_id>
```